nextflow.config

```diff
@@ -24,39 +157,55 @@ params {
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
+//  igenomes_base = 's3://ngi-igenomes/igenomes/'                                                   // pipeline does not support igenomes
   tracedir = "${params.outdir}/pipeline_info"
-  igenomes_ignore = false
-  custom_config_version = 'master'
-  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-  hostnames = false
-  config_profile_description = false
-  config_profile_contact = false
-  config_profile_url = false
-
+//  igenomes_ignore = false                                                                         // pipeline does not support igenomes
   // Defaults only, expecting to be overwritten
   max_memory = 128.GB
   max_cpus = 16
   max_time = 240.h
 
 }
 
-// Container slug. Stable releases should specify release tag!
-// Developmental code should specify :dev
+
+//--------
+// Work directory:
+//--------
+workDir = "$baseDir/work"
+
+
+//--------
+// Container:
+//--------
 process.container = 'nfcore/dualrnaseq:dev'
 
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
 
+
 // Load nf-core custom profiles from different Institutions
 try {
   includeConfig "${params.custom_config_base}/nfcore_custom.config"
 } catch (Exception e) {
   System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
 }
 
+// Load nf-core/dualrnaseq custom profiles from different Institutions
+try {
+  includeConfig "${params.custom_config_base}/pipeline/dualrnaseq.config"
+} catch (Exception e) {
+  System.err.println("WARNING: Could not load nf-core/config/dualrnaseq profiles: ${params.custom_config_base}/pipeline/dualrnaseq.config")
+}
```

Comment:
No custom profile for dualrnaseq created yet.
Do you plan to make/use one for your institute?

Reply:
We have been talking about this, but haven't come to a conclusion. @bozmik and I are in communication with IT about potentially being able to run Docker as well as Singularity - we are still working out how this will work re permissions. The short answer is - we are working on it :-)
Until we have more info from IT - would you recommend commenting out these lines?

Reply:
Not at all, but if you do intend to keep these lines, then you should at least add the proper files in the configs repo too.
cf: https://github.com/nf-core/configs#adding-a-new-pipeline-specific-config

But running docker and or singularity is something that you can do just at the institutional level, there's no need to do a pipeline-institututional specific config for that

Reply:
As we can add a pipeline-specific institutional config file instead of the main institutional config, I think it would be useful to provide such a file. We will try to add it soon.

Reply:
ping me when you add that, because I'm still unsure what you intend to do with that.
I'm afraid this might be overkill.

Reply:
I agree with you @MaxUlysse - will resolve this and focus on the comment below